### PR TITLE
Fix various missing option help messages ...

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
A pack of options for cms stay undocumented ...

### CODE REVIEW

File: apps/ocsp.c [hunk 1]
```diff
@@ -124,7 +125,7 @@ OPTIONS ocsp_options[] = {
     {"no_chain", OPT_NO_CHAIN, '-', "Don't chain verify response"},
     {"no_cert_checks", OPT_NO_CERT_CHECKS, '-',
      "Don't do additional checks on signing certificate"},
-    {"no_explicit", OPT_NO_EXPLICIT, '-'},
+    {"no_explicit", OPT_NO_EXPLICIT, '-', "OCSP_NOEXPLICIT"},
```
Comment (richsalz) [inline apps/ocsp.c]: Do not explicitly check the chain, just verify the root.

File: apps/smime.c [hunk 2]
```diff
@@ -88,14 +88,15 @@ OPTIONS smime_options[] = {
...
-    {"nooldmime", OPT_NOOLDMIME, '-'},
-    {"crlfeol", OPT_CRLFEOL, '-'},
+    {"resign", OPT_RESIGN, '-', "Resign a signed message"},
+    {"nochain", OPT_NOCHAIN, '-', 
+     "set PKCS7_NOCHAIN so certificates contained in the message are not used as untrusted CAs" },
+    {"nosmimecap", OPT_NOSMIMECAP, '-', "Disable use of SMIMECapabilities attribute"},
+    {"stream", OPT_STREAM, '-', "Enable CMS streaming" },
+    {"indef", OPT_INDEF, '-', "Same as -stream" },
+    {"noindef", OPT_NOINDEF, '-', "Disable CMS streaming"},
+    {"nooldmime", OPT_NOOLDMIME, '-', NULL},
```
Comment (richsalz) [inline apps/smime.c]: these seems to be a no-op now; we should remove it.

File: apps/cms.c [hunk 3]
```diff
@@ -122,22 +122,22 @@ OPTIONS cms_options[] = {
...
     {"nosigs", OPT_NOSIGS, '-', "Don't verify message signature"},
     {"no_content_verify", OPT_NO_CONTENT_VERIFY, '-'},
     {"no_attr_verify", OPT_NO_ATTR_VERIFY, '-'},
-    {"stream", OPT_INDEF, '-'},
-    {"indef", OPT_INDEF, '-'},
-    {"noindef", OPT_NOINDEF, '-'},
+    {"stream", OPT_INDEF, '-', "Enable CMS streaming"},
+    {"indef", OPT_INDEF, '-', "Same as -stream"},
+    {"noindef", OPT_NOINDEF, '-', "Disable CMS streaming"},
     {"nooldmime", OPT_NOOLDMIME, '-'},
```
Comment (richsalz) [inline apps/cms.c]: seems to be a no-op, we should remove it.

File: apps/smime.c [hunk 4]
```diff
@@ -88,14 +88,15 @@ OPTIONS smime_options[] = {
...
-    {"nosmimecap", OPT_NOSMIMECAP, '-'},
-    {"stream", OPT_STREAM, '-'},
-    {"indef", OPT_INDEF, '-'},
-    {"noindef", OPT_NOINDEF, '-'},
-    {"nooldmime", OPT_NOOLDMIME, '-'},
-    {"crlfeol", OPT_CRLFEOL, '-'},
+    {"resign", OPT_RESIGN, '-', "Resign a signed message"},
+    {"nochain", OPT_NOCHAIN, '-', 
+     "set PKCS7_NOCHAIN so certificates contained in the message are not used as untrusted CAs" },
+    {"nosmimecap", OPT_NOSMIMECAP, '-', "Disable use of SMIMECapabilities attribute"},
```
Comment (richsalz) [inline apps/smime.c]: see PKCS7_sign.pod; not sure about the wording of this.

**Comment by FdaSilvaYY** [comment]
Fixed, even if GH hasn't fold your comments.

**Comment by richsalz** [comment]
bugfix.  for master and 1.1.0 +1

**Comment by dot-asm** [comment]
+1

**Comment by richsalz** [comment]
merge to master and 1.1.0, thanks!

**Pull request closed without merging**